src/getLocalesPaths.js

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-console */
+import fs from "fs"
+import path from "path"
+import yaml from "js-yaml"
+import glob from "glob"
+
+const configFilePath = ".phraseapp.yml"
+
+const getLocaleGlobs = () => {
+  try {
+    const phraseapConfig = yaml.safeLoad(fs.readFileSync(path.join(process.cwd(), configFilePath), "utf8"))
+    return phraseapConfig.phraseapp.pull.targets.map(target => {
+      return target.file.replace("<locale_name>", "*")
+    })
+  } catch {
+    console.error("Phraseapp config not found falling back to static/locales/*/*.json")
+
+    return [ "static/locales/*/*.json" ]
+  }
+}
+
+export default () => {
+  const localesGlobs = getLocaleGlobs()
```

Comment:
In which case would there be multiple glob strings in the array? Isnt it always one?

Reply:
If we ever get to support multiple locale files and multiple namespaces we would end up with multiple entries in the file config. Or at least I think so. Since the effort of supporting it is actually lower than not I'd rather do it now than need to refactor the code later.